linguatec_lexicon/views.py

```diff
@@ -75,11 +79,182 @@ def validate(self, xlsx_file):
         return out
 
 
+class ImportDataView(TemplateView):
+    template_name = "linguatec_lexicon/importdata.html"
+
+    def create_imports_info(self, input_file, type):
+        ii = ImportsInfo.objects.create(status='created', input_file=input_file, type=type)
+        return ii.pk
+
+    def post(self, request, *args, **kwargs):
+        context = self.get_context_data(**kwargs)
+
+        type_import = str(request.POST.get('type_import'))
+
+        if type_import == 'data' or type_import == 'variation':
```

Comment:
Please define constants instead of hardcode `data`, `variation`... 
Maybe you can define it on the model ImportsInfo (ImportLog) and then access to it outside using something like: `ImportLog.TYPE_DATA`

You can check [django-importer](https://github.com/ribaguifi/django-importer/blob/master/djimporter/models.py#L6-L16) to get some inspiration.

Reply:
Corrected, now is used the constants already defined instead of hardcoding them each time, thank you.